docs/guides/updating-state.md

```diff
@@ -10,6 +10,9 @@ the new state, and it will be shallowly merged with the existing state in the
 store. **Note** See next section for nested state.
 
 ```tsx
+import { create } from 'zustand'
```

Comment:
I just realized that this causes a deprecation error. It should be `import { createWithEqualityFn } from 'zustand/traditional'`.

Now, I don't think the purpose of this doc is about shallow, so how about changing like this?

```diff
-   // "select" the needed state and actions, in this case, the firstName value
-   // and the action updateFirstName
-   const [firstName, updateFirstName] = useStore(
-     (state) => [state.firstName, state.updateFirstName],
-     shallow
-   )
+   const firstName = useStore((state) => state.firstName)
+   const updateFirstName = useStore((state) => state.updateFirstName)
```

Also, we should probably change `useStore` to `useNameStore` or something to avoid confusion with exported useStore hook.

Reply:
Ok, so you want I remove the `import shallow` and also `shallow` word in useStore function in App component? As in the scrrenshot?

Reply:
I think `shallow` doesn't make much sense for this doc. So, in the end, it should be:
```js
const firstName = useNameStore((state) => state.firstName)
const updateFirstName = useNameStore((state) => state.updateFirstName)
```
Or maybe `usePersonStore`.

Reply:
> so how about changing like this?

This is what I suggested in the next screenshot
To remove the shallow, see the second section-

<img width="722" alt="צילום מסך 2023-09-06 134330" src="https://github.com/pmndrs/zustand/assets/114335827/6d85658a-2da1-4f2f-943d-5ffe5983062c">



So what I need to do now with `shallow` and `import shallow`?